lagunita/lms/templates/instructor/instructor_dashboard_bottom.html

```diff
@@ -0,0 +1,8 @@
+<script language="JavaScript" type="text/javascript">
```

Comment:
language attribute has been deprecated for a while and should not be used: https://stackoverflow.com/questions/2267476/html-script-tag-type-or-language-or-omit-both